http-server/src/test/java/io/airlift/http/server/TestHttpServerConfig.java

```diff
@@ -111,6 +112,7 @@ public void testExplicitPropertyMappings()
                 .put("http-server.http2.session-receive-window-size", "4MB")
                 .put("http-server.http2.stream-receive-window-size", "4MB")
                 .put("http-server.http2.input-buffer-size", "4MB")
+                .put("http-server.http2.stream-idle-timeout", "20s")
```

Comment:
It's usually better to use a non-round value (random-ish) like `23s` that wouldn't ever be used as a default.